angular/polyfills-es5.js

```diff
@@ -1 +1 @@
\ No newline at end of file
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Zone' is not defined.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5486792)